app/javascript/controllers/transcript_controller.js

```diff
@@ -89,6 +89,8 @@ export default class extends Controller {
     // changing disabled mode to "hidden" to prevent losing cue information.
     if(videojs.browser.IS_ANY_SAFARI) {
       this.convertDisabledTracks()
+      // track.cues returns 0 in safari, even when there are cues
```

Comment:
I'm not sure why this is relevant.  Is there more to context you can pass on here?

Reply:
Sure. I can update the comment, the long and short is if there are two vtt files with captions, return captions.filter(track => this.trackCues(track).length) returns a list of one file. This has to do with something weird in safari that returns track.cues.length == 0, on hidden tracks.